### DISCUSSION TITLE
What is the CSS variable name for single brackets around text [text]

### DISCUSSION THREAD

**Discussion by BerylTP** [AnubisNekhet/AnuPpuccin#221] [Snippets]
I have a number of areas and quotes where the single bracket is used. On some themes these are given a colour. I would like to specify a colour for these, what are they called though? e.g. table, tag, link, file etc

**Comment by syrth-io** [comment]
after a quick look via inspector, it looks like the class name is `.cm-s-obsidian span.cm-formatting-link`:

```css
.cm-s-obsidian span.cm-formatting-link {
    color: var(--text-faint);
}
```
however, you may need to use additional selectors (perhaps start with adding `span` to `.cm-s-obsidian`). I have not tested it (though I might soon, as I also like this effect!), so I can't say for certain without deeper digging (also note that `cm-hmd-barelink` does not appear to be defined in the native CSS despite appearing in the span class). this is the HTML in the live preview code:

```html
<span class="cm-formatting cm-formatting-link cm-hmd-barelink cm-link" spellcheck="false">[</span>
<span class="cm-hmd-barelink cm-link">test</span>
<span class="cm-formatting cm-formatting-link cm-hmd-barelink cm-link" spellcheck="false">]</span>
```

the code is identical both with the native theme in a fresh vault and with anuppuccin installed (albeit with no settings changed). another option would be to change the color of the `--text-faint` variable but that would then change any element with that color, and there may be other variables dependent on the value, so it could just be more work in the end.

unfortunately, it doesn't look like there's native support for it in reading mode, and I can't think off the top of my head how you might achieve it via CSS as there's no selectors at all:
![screenshot of HTML code, showing an opening div element followed by an opening p, and then a line of plain text reading "bare link test", with the word test enclosed in brackets. a br tag is on the last line.](https://github.com/AnubisNekhet/AnuPpuccin/assets/112804375/62d392fd-d3c7-40ec-895a-45c4e302c7ba)

if I end up testing it, I'll come back and update/reply to this!
